prow/github/reporter/reporter.go

```diff
@@ -55,6 +57,11 @@ func (c *Client) ShouldReport(pj *v1.ProwJob) bool {
 		return false
 	}
 
+	if c.buildOnly && pj.Spec.Agent != v1.KnativeBuildAgent {
```

Comment:
No need to tie this so tightly to knative-build (we'll probably have a pipeline agent next quarter, who knows what else)

How about: `if c.agentFocus != "" && pj.Spec.Agent != c.agentFocus`

Reply:
missed the notification 😂will address tomorrow

Reply:
thanks!

Reply:
updated :-)